### PR TITLE
Add various misc improvements for ACE Arsenal

### PR DESCRIPTION
**Pull request by alganthe**
**When merged this pull request will:**
- Disable animations when the center isn't the player
- Change the edit loadout button in 3DEN to be a menu with 2 sub-entries, one being BI VA the other ACE Arsenal
- Revert changes to "add full arsenal" and "remove arsenal" zeus modules, create dedicated ones for ACE Arsenal
- Add 3 new arsenal EHs
- Add a setting to disable the identity tabs.
- Resize the right panel to not have empty space when the "load" bar isn't present
- Added a tooltip for the clear container button
- Added classname copying of the current selection for the left and right panel

<details>
  <summary>New EHs click to expand</summary>

| Name  | Arguments |
| ------------- | ------------- |
| ace_arsenal_onLoadoutLoad | loadout data (ARRAY) || ace_arsenal_loadoutImported | Arsenal display (DISPLAY), (import list (BOOL) |
| ace_arsenal_loadoutImported | Arsenal display (DISPLAY), (import list (BOOL) |
| ace_arsenal_loadoutExported | Arsenal display (DISPLAY), export list (BOOL) |
</details>

### CODE REVIEW

File: addons/arsenal/functions/fnc_onArsenalOpen.sqf [hunk 1]
```diff
@@ -267,6 +267,19 @@ if (is3DEN) then {
...
     _buttonCloseCtrl ctrlSetText (localize "str_ui_debug_but_apply");
+} else {
+    GVAR(centerNotPlayer) = (GVAR(center) != player);
+
+    {
+        private _ctrl = _display displayCtrl _x;
+        _ctrl ctrlEnable GVAR(enableIdentityTabs);
+        _ctrl ctrlSetFade ([0.6, 0] select GVAR(enableIdentityTabs));
+        _ctrl ctrlcommit 0;
+    } foreach [
```
Comment (Dystopian) [inline addons/arsenal/functions/fnc_onArsenalOpen.sqf]: ctrlcommit, foreach -> ctrlCommit, forEach

**Comment by bux** [comment]
`ace_arsenal_onLoadoutLoad` totally needs the name of the loadout passed too

**Comment by alganthe** [comment]
^ Done.

**Comment by alganthe** [comment]
Should be good to go.

**Pull request merged**